src/proposals/proposalTypes/HybridProposal.sol

```diff
@@ -551,8 +624,105 @@ abstract contract HybridProposal is
+                "WORMHOLE_CORE_BASE",
+                baseChainId
+            );
+            address wormholeCoreBaseSepolia = addresses.getAddress(
+                "WORMHOLE_CORE_SEPOLIA_BASE",
+                baseSepoliaChainId
+            );
+            address temporalGov = addresses.getAddress(
+                "TEMPORAL_GOVERNOR",
+                sendingChainIdToReceivingChainId[block.chainid]
+            );
+            address wormholeCoreMoonbeam = block.chainid == moonBeamChainId
+                ? addresses.getAddress(
+                    "WORMHOLE_CORE_MOONBEAM",
+                    moonBeamChainId
+                )
+                : addresses.getAddress(
+                    "WORMHOLE_CORE_MOONBASE",
+                    moonBaseChainId
+                );
+
+            address[] memory targets = new address[](baseActions.length);
+            uint256[] memory values = new uint256[](baseActions.length);
+            bytes[] memory calldatas = new bytes[](baseActions.length);
+
+            for (uint256 i = 0; i < baseActions.length; i++) {
+                targets[i] = baseActions[i].target;
+                values[i] = baseActions[i].value;
+                calldatas[i] = baseActions[i].data;
+            }
+
+            /// assert wormhole core BASE address is not in the list of targets on Moonbeam
+            for (uint256 i = 0; i < targets.length; i++) {
+                require(
+                    targets[i] != wormholeCoreBase,
+                    "Wormhole Core BASE address should not be in the list of targets"
+                );
+                require(
+                    targets[i] != wormholeCoreBaseSepolia,
+                    "Wormhole Core BASE Sepolia address should not be in the list of targets"
+                );
+            }
+
+            for (uint256 i = 0; i < baseActions.length; i++) {
+                /// there's 0 reason for any proposal actions to target wormhole core on base
+                require(
+                    baseActions[i].target != wormholeCoreMoonbeam,
+                    "Wormhole Core Moonbeam address should not be in the list of targets for Base"
+                );
+                require(
+                    baseActions[i].target != wormholeCoreBase &&
+                        baseActions[i].target != wormholeCoreBaseSepolia,
+                    "Wormhole Core Base address should not be in the list of targets for Base"
+                );
+
+                if (baseActions.length >= 2 && i < baseActions.length - 2) {
+                    require(
+                        baseActions[i].target != wormholeCoreBase,
+                        "Wormhole Core BASE address should be the last target for Base"
+                    );
+                }
+            }
+
+            for (uint256 i = 0; i < moonbeamActions.length; i++) {
+                require(
+                    moonbeamActions[i].target != temporalGov,
```

Comment:
lets put this together with the loop that asserts wormhole core BASE address is not in the list of targets on Moonbeam